### ISSUE TITLE
LinkControl: Add blank icon for custom post type menu items

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#55593]
Instead of omitting the icon entirely, the search result title and URLs should always be aligned vertically. This way it’s easier to scan the results to find the correct item. 

We should add a blank icon to wordpress/icons that can be used in cases like this (and the command palette). 

![Portfolio 1](https://github.com/WordPress/gutenberg/assets/1813435/3feb7c58-93d4-4637-a82f-e41621f48d10)

**Comment by vcanales** [comment]
I see a `customPostTypes` icon in the library; is that a good fit instead of a blank one? 


![Image](https://github.com/WordPress/gutenberg/assets/1157901/b7a34e68-f462-4949-a2d4-d0f2ba998484)

**Comment by richtabor** [comment]
I still think a blank icon would work better perhaps. We do the same in the command palette (but manually spaced with CSS): 

![CleanShot 2023-10-25 at 11 16 19](https://github.com/WordPress/gutenberg/assets/1813435/a284870f-9a59-4ede-bbdf-2cd45597089b)

**Comment by vcanales** [comment]
I've found that custom post types don't show up in the suggestions for the Link Control, since Gutenberg v16.8.0 — I haven't been able to track down the reason...


16.8.0
![Image](https://github.com/WordPress/gutenberg/assets/1157901/1131c4ca-07ef-40b7-ab85-502c1b420163)

16.7.0
![Image](https://github.com/WordPress/gutenberg/assets/1157901/7c4eefb2-b078-421f-a299-8f94a87daf2a)

**Comment by richtabor** [comment]
Did you add a plugin with a custom post type, like https://wordpress.org/plugins/portfolio-post-type/?

**Comment by vcanales** [comment]
Yeah, the videos above are the same WordPress instance, with different Gutenberg versions loaded, with a couple of custom types added through the CPT UI Plugin.

**Comment by BugReportOnWeb** [comment]
> I've found that custom post types don't show up in the suggestions for the Link Control, since Gutenberg v16.8.0 — I haven't been able to track down the reason...

As of Gutenberg 21.6.0, I noticed that it can been seen when choosing to add a custom link (doesn't show on default which I assume only shows the pages)

<img width="443" height="280" alt="Image" src="https://github.com/user-attachments/assets/e677d719-4848-46be-b40e-5c4841d60563" />

@vcanales Are you still working on this? If not, I have a working patch ready and would love to create a PR

**Comment by vcanales** [comment]
> [@vcanales](https://github.com/vcanales) Are you still working on this? If not, I have a working patch ready and would love to create a PR

@BugReportOnWeb feel free to take over
